src/promptflow/promptflow/_cli/_pf_azure/_flow.py

```diff
@@ -89,28 +101,27 @@ def add_parser_flow_download(subparsers):
     )
 
 
+def dispatch_flow_commands(args: argparse.Namespace):
+    if args.sub_action == "create":
+        create_flow(args)
+
+
 def _get_flow_operation(subscription_id, resource_group, workspace_name):
     pf_client = _get_azure_pf_client(subscription_id, resource_group, workspace_name)
     return pf_client._flows
 
 
-def create_flow(
-    source: Union[str, PathLike, IO[AnyStr]],
-    workspace_name: str,
-    resource_group: str,
-    subscription_id: str,
-):
+def create_flow(args: argparse.Namespace):
     """Create a flow for promptflow."""
-    source = Path(source).resolve().absolute()
-    if not source.exists():
-        raise FileNotFoundError(f"Source folder {str(source)!r} does not exist.")
-    if not source.is_dir():
-        raise ValueError(f"Source path {str(source)!r} should be a folder, got a file.")
-    flow_entity = load_flow(source)
-    flow_operations = _get_flow_operation(subscription_id, resource_group, workspace_name)
-    flow_draft = flow_operations._create_or_update(flow_entity)
-    print(f"Successfully created flow {flow_draft.flow_name!r} with flow id {flow_draft.flow_id!r}.")
-    return flow_draft
+    pf = _get_azure_pf_client(args.subscription, args.resource_group, args.workspace_name)
+    params = _parse_flow_metadata_args(args.params_override)
+    pf.flows.create_or_update(
+        source=args.source,
+        flow_name=params.get("name", None),
+        flow_type=params.get("type", None),
+        description=params.get("description", None),
+        tags=params.get("tags", None),
+    )
```

Comment:
what's the output of pfazure flow create?

Reply:
Currently is a flow dict that contains flow's metadata, see screenshot in the summary.